### PR TITLE
balance stats

### PR DESCRIPTION
**Pull request by jonathaneeckhout**
Tackles:
https://github.com/jonathaneeckhout/jdungeon/issues/66

partially. Game testing will probably refine the stats more.

Comments:
- I've implemented 1 attack power per 5 levels instead of increasing 1 attack per level

### CODE REVIEW

File: scenes/enemies/MoldedDruvar/MoldedDruvar.gd [hunk 1]
```diff
@@ -50,7 +50,8 @@ func _ready():
 
 
 func _add_loot():
-	add_item_to_loottable("Gold", 25, 100)
+	add_item_to_loottable("Gold", 0.75, 300)
+	add_item_to_loottable("Club", 0.05, 1)
```
Comment (ivoryfitz) [inline scenes/enemies/MoldedDruvar/MoldedDruvar.gd:54]: Gonna sound silly, but thematically speaking, should it be an axe that has a chance to drop?

Comment (ivoryfitz) [reply scenes/enemies/MoldedDruvar/MoldedDruvar.gd:54]: Oh wait, the Axe is like god-tier strong in comparison, nevermind.

Comment (jonathaneeckhout) [reply scenes/enemies/MoldedDruvar/MoldedDruvar.gd:54]: also the loot isn't really sorted out in general

**Comment by jonathaneeckhout** [comment]
Going to merge this one, play testing will refine it more.

**Pull request merged**